src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -110,7 +110,28 @@ paths:
                 type: integer
                 format: int32          
               
-              
+ 
+  /recordTreeLocal/{id}:
+    get:
+      tags:
+        - '${project.name}'
+      summary: 'Return the record with local record tree. The local record will only has parent and children loaded as object. The parent object reference will be loaded and so will the children from this record. Will return throw exception if no record with the given ID exists. A record marked with delete flag will also be returned.'
+      operationId: getRecordTreeLocal
+      parameters:
+        - name: id
+          in: path
+          description: 'Record ID'
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'The DsRecord'
```

Comment:
Maybe distinguish it from the `getRecord`-response? `The DsRecord including immediate parent and children records`.

Reply:
Good formulation, accepted.